modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -341,6 +341,7 @@
 				# a is -3
 				a = floor(-2.99)
 				[/codeblock]
+				[b]Note:[/b] This method returns a float. If you need an integer, you can use [code]int(s)[/code] directly.
```

Comment:
Maybe also change the codeblock so that the comments use `2.0` and `-3.0`?

Reply:
@akien-mga Good catch :slightly_smiling_face: